### PR TITLE
New path handler

### LINKED ISSUE #22: Improve path handling
Current path accessing is thrash, should be updated.

### PR DESCRIPTION
**Pull request by GitThisJoel**
You like accessing files? Me too! Now it can be done more easily! :^)

Closes #22

### CODE REVIEW

File: main/helpers/path_handler.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import sys, os
```
Comment (erik-stein) [inline main/helpers/path_handler.py:1]: These unused imports can be removed

File: main/tex_compiler/tex_compiler.py [hunk 2]
```diff
@@ -96,10 +98,9 @@ def _beskrivningfordelning_fordelning_sum(self, sales):
 
     def _prepend_commands(self):
         cmds = str(self.tex_commands)
+        with open(self.intakt_skeleton, "r") as skel_f:
+            tex_skeleton = skel_f.read()
```
Comment (erik-stein) [inline main/tex_compiler/tex_compiler.py:102]: An alternative way to read (and write) files with pathlib:
```suggestion
        tex_skeleton = self.intakt_skeleton.read_text()
```
Which do you think is better?

(If we decide to use read_text and write_text maybe it is better to change it everywhere in a separate pull request)

**Review by erik-stein — CHANGES_REQUESTED** [review]
Very nice!

`main/helpers/asset_loader.py` should probably be changed too.

File: main/tex_compiler/tex_compiler.py (hunk 2 above)
Comment (GitThisJoel) [reply main/tex_compiler/tex_compiler.py:102]: I agree that it looks better, might be better to open a new issue and do it in another PR.

Comment (GitThisJoel) [reply main/tex_compiler/tex_compiler.py:102]: I opened #22 for this containing this information.

**Comment by GitThisJoel** [comment]
@erik-stein do you approve of my reasoning?

File: main/helpers/asset_loader.py [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
 import json
+from path_handler import main_dir
 
 
 class AssetLoader:
     def __init__(self):
-        self._asset_directory = (
-            os.path.dirname(os.path.realpath(inspect.getfile(self.__class__))) + "/../assets/"
-        )
+        self._asset_directory = main_dir() / "assets"
```
Comment (erik-stein) [inline main/helpers/asset_loader.py:7]: You have `assets_dir()` in `path_handler` which I guess was intended for this class.
```suggestion
        self._asset_directory = assets_dir()
```

Or this attribute could be removed altogether and replaced with `assets_dir()` in `load_assets`

**Review by erik-stein — CHANGES_REQUESTED** [review]
One last suggestion, then it gets my approval :)

I agree that it looks better without the `with open...` "boilerplate", I might look into that issue later.

**Comment by GitThisJoel** [comment]
If you approve, @erik-stein, I will merge 😁

**Review by erik-stein — APPROVED** [review]

**Pull request merged**